module/learning-resources/helper.js

```diff
@@ -0,0 +1,98 @@
+/** 
+* name : helper.js
+* author : Rakesh Kumar
+* created-date : 02-Nov-2020
+* Description : Related to learning resources
+*/
+
+// Dependencies
+const sunbirdService = require(ROOT_PATH + "/generics/services/sunbird-application");
```

Comment:
@rakeshSgr - Why not use the existing sunbird service?

Reply:
sunbird.js is pointing to https://dev.bodh.shikshalokam.org" and sunbird-application.js pointing to our sunbird service application, so I kept different.